### DISCUSSION TITLE
The openDTU's "data age" continuously increases to minutes

### DISCUSSION THREAD

**Discussion by fkroepfl** [tbnobody/OpenDTU#740] [Q&A] [answered]
The openDTU's "data age" continuously increases to minutes even though the openDTU and the inverter are in 2m line of sight.

My setup consists of original Hoymiles components and most recently an openDTU:

Inverter Model: HM-1200	
- Hardware: Ver.H00.04.00	
- Software: Ver.V01.00.16

DTU-W100
- Hardware Ver.H07.01.01
- Software Ver.V00.02.08

OpenDTU-E996D8
- SDK version: v4.4.3
- Config version: 0.1.24
- Firmware version / Git hash: 055eb3a

Has anyone experienced this problem before?
Any idea how to fix the problem?

Sunny greetings,
/FJ

**Comment by Solarteur** [comment, marked as answer]
1. If I remember correctly, either OpenDTU or the original Hoymiles DTU should be running, not both in parallel. This is easy to test.
2. The nRF24L01+ module may be defective, replace it and test again.
3. For the sake of completeness: The inverter only sends data when it is running/feeding.

**Reply by fkroepfl** [reply]
@Solarteur 

ad 1) Thanks for the tip. I will try that right away and deactivate the HM-DTU. 

btw: Is it an absolute must or a recommendation that either OpenDTU or the original Hoymiles-DTU runs, but not both in parallel or does that apply temporarily or forever?

Background of the question is simply that I wanted to continue using Hoymiles Cloud with all dashboards - at least until I have an alternative for the most important ones. Quasi I wanted to do a gradual migration ;-)

ad 2) if 1) doesn't work out, I guess I will have to contact my OpenDTU supplier ;-)

ad 3) even if I had read that originally in the manual, I had forgotten it again and was a bit irritated on the first evening, because "Live View" does not simply show the status for "Reachable": "Off" as with MQTT, but instead the data age is increased.

Thanks again,
/FJ

**Reply by fkroepfl** [reply]
@Solarteur 
My OpenDTU HW provider told me that he himself runs several OpenDTUs simultaneously with one and the same inverter without any problems. However, he does not have a HM DTU, but this should not have any influence according to my understanding.

I have now deactivated my HM DTU for a longer period of time and had to realize that nothing changes in the behavior of the OpenDTU, i.e. the errors still occur, and in my opinion too frequently.

Next steps: 
- Test with another new inverter, a HM-1500.
- Replacement of the OpenDTU HW, which inevitably entails a new nRF24L01+ module.

/FJ

**Comment by fkroepfl** [comment]
![image](https://user-images.githubusercontent.com/6333880/227711029-d55dea15-596f-4eb1-8256-3607a25e0e07.png)
The accessibility of my OpenDTU captured via MQTT/Home Assistant as additional info. At first glance, parallel operation should not actually be the cause of the instabilities.

**Comment by tbnobody** [comment]
siehe https://github.com/tbnobody/OpenDTU/issues/727, https://github.com/tbnobody/OpenDTU/discussions/300

**Reply by fkroepfl** [reply]
@tbnobody OK, thanks for pointing out that you can communicate with the inverter only through a single device. 🤗

But what I still do not understand in my case, where the instability in communication comes from, even if only the OpenDTU communicates with the inverter as the only DTU? ([please see the screenshot in the previous post](https://user-images.githubusercontent.com/6333880/227711029-d55dea15-596f-4eb1-8256-3607a25e0e07.png))
